.ci/azure-pipelines.yml

```
@@ -78,15 +78,22 @@ jobs:
         env:
           BINTRAY_KEY: $(bintray_key)
 
+      - task: CopyFiles@2
+        displayName: 'Copy build assets'
+        inputs:
+          Contents: |
+            **/distributions/*-$(JELLYFIN_VERSION).zip
+            **/libs/*-$(JELLYFIN_VERSION).jar
+            **/libs/*-$(JELLYFIN_VERSION)-sources.jar
+            **/outputs/aar/*.arr
```

I think this should be

```suggestion
            **/outputs/aar/*.aar
```
:wink:

---

yes :facepalm: :grin: